packages/headless/src/controllers/product-listing/headless-product-listing.ts

```diff
@@ -25,6 +28,50 @@ const optionsSchema = new Schema({
   }),
 });
 
+
+export enum SortBy {
```

Comment:
Quite a bit of duplication from stuff that exists elsewhere in the project, that I think we could just refactor a bit and re-use properly.